### DISCUSSION TITLE
Main file

### DISCUSSION THREAD

**Discussion by dheeraj1129** [CatsInTech/GitHub-ReadMe#43] [General]
which one is the main file and where it is??

**Comment by praveenscience** [comment]
Hi @dheeraj1129, what do you mean by main file? Which one are you talking about? Did you read the ReadMe completely for the installation and running instructions? Please let me know if you know how to work with React JS?

Kindly answer all my questions so that I can guide you better. 😊

**Reply by dheeraj1129** [reply]
Hello @praveenscience  I successfully clone the projects file by GitHub desktop, and  i see some issue on website and i think i can fix it, majorly CSS part,  but i am struggling to find React , Html, CSS, JS,  files. Now i am going to read the ReadMe file again..

**Reply by praveenscience** [reply]
Sure man... Check once...

**Reply by dheeraj1129** [reply]
So I readied, can you tell me is git installation  is mandatory. Can I do that with GitHub desktop( it works on other projects as well ) but not here.  Because i have some problem with git installation.

**Reply by praveenscience** [reply]
yea, @dheeraj1129, the whole process uses Git for working on these. You can try without Git or GitHub for the same too, but it's a bit difficult and convoluted.
